server/routerlicious/kubernetes/routerlicious/values.yaml

```diff
@@ -68,7 +68,9 @@ zookeeper:
   url: zookeeper_url:port
 
 mongodb:
-  url: mongodb_url
+  operationsDbEndpoint: mongodb_url
```

Comment:
@curtisman Before this is checked in, we need to update the r11s deployment config.

Reply:
Copying from offline discussion, change merged by curtis here: https://dev.azure.com/fluidframework/internal/_git/ff_internal/pullrequest/45